### PR TITLE
Chino/eme 281 add author to cards

### PR DESCRIPTION
**Pull request by IgnacioDebat**

### CODE REVIEW

File: src/lib/components/cards/ContentCard.svelte [hunk 1]
```diff
@@ -129,6 +147,53 @@
...
+				align-items: center;
+				gap: var(--space-1);
+				color: var(--clr-text-main);
+				margin-bottom: var(--space-3);
+				font-size: 0.82rem;
+				.tooltip-wrapper {
+					position: relative;
+					display: flex;
+
+					.tooltip-text {
```
Comment (mateoroldos) [inline src/lib/components/cards/ContentCard.svelte]: @IgnacioDebat we already have a tooltip element in the component library!

Here is an example of how to implement it: https://github.com/emerald-dao/component-library/blob/f3dc2937a5e4dc834ea9be81c598a6ce58500a92/src/lib/components/TooltipIcon/TooltipIcon.svelte#L11

You just need to add `data-tooltip: "<the text you want>"` inside any HTML tag!

**Review by mateoroldos — CHANGES_REQUESTED** [review]
Small change request: We already have a tooltip implementation on the component library..
It'll be better to use that one to maintain design standards an make code shorter!

File: src/lib/components/cards/ContentCard.svelte (hunk 1 above)
Comment (jacob-tucker) [reply src/lib/components/cards/ContentCard.svelte:162]: I will work on extending this out to be used by any Icon :)

Comment (jacob-tucker) [reply src/lib/components/cards/ContentCard.svelte]: https://github.com/emerald-dao/emerald-academy-v2/blob/c7092dec3762110f09a505bab2cb2b6a8a07435e/src/lib/components/cards/ContentCard.svelte#L53-L60

Here is a good way to do it @IgnacioDebat. You can see just by using `data-tooltip`, you can include the tooltip without having to make a separate span :) I hope this helps! And sorry you had to style so much without knowing data-tooltip 😆

Comment (IgnacioDebat) [reply src/lib/components/cards/ContentCard.svelte]: Thank you @jacob-tucker ! After talking with Mateo, we decided that the best thing to do was to centralize in a new component all the places where the Author appears. In this new component I used the tooltip of the component library. Now in all the places where the Author appears, it appears as follows: 

image

**Review by IgnacioDebat — COMMENTED** [review]
The component library tooltip element is the one we are using now

**Pull request merged**